plrust/src/user_crate/crating.rs

```diff
@@ -308,35 +311,32 @@ fn unsafe_mod(mut called_fn: syn::ItemFn, variant: &CrateVariant) -> eyre::Resul
     .wrap_err("Could not create opened module")
 }
 
-fn safe_mod(bare_fn: syn::ItemFn) -> eyre::Result<syn::ItemMod> {
+fn safe_mod(bare_fn: syn::ItemFn) -> eyre::Result<(syn::ItemMod, LintSet)> {
     let imports = shared_imports();
-    // Hello from the futurepast!
-    // The only situation in which you should be removing this
-    // `#![forbid(unsafe_code)]` is if you are moving the forbid
-    // command somewhere else  or reconfiguring PL/Rust to also
-    // allow it to be run in a fully "Untrusted PL/Rust" mode.
-    // This enables the code checking not only for `unsafe {}`
-    // but also "unsafe attributes" which are considered unsafe
-    // but don't have the `unsafe` token.
-    syn::parse2(quote! {
+    let lints = compile_lints();
+
+    let code = syn::parse2(quote! {
+        #[forbid(unknown_lints)]
```

Comment:
Hm. I think we should actually allow users to write code that applies an unknown lint _inside their own function_, which means letting them use `#[allow(unknown_lints)]` inside the item. It would be... weird, frankly, to do this, but it's not a security issue because the list of _our_ lints inside the module will still be checked, even if it's later overridden.
```suggestion
        #[deny(unknown_lints)]
```

Reply:
Alrighty, that's cool